src/views/CampusSafety/views/LostAndFound/index.tsx

```diff
@@ -362,54 +393,96 @@ const LostAndFound = () => {
 
       {titleCard()}
 
-      {/* Active Missing Item Reports */}
-      <Grid container justifyContent="center" spacing={3} marginTop={3}>
-        <Grid item xs={12} md={10}>
-          <Card>
-            <CardHeader
-              className="gc360_header"
-              title={
-                <Typography variant="h5" align="center">
-                  My Active <span className={styles.yellowText}>Lost</span> Item Reports
-                </Typography>
-              }
-            />
-          </Card>
+      {/* Recently Found Reports */}
+      {foundReports.length > 0 && (
+        <Grid container justifyContent="center" spacing={3} marginTop={3}>
+          <Grid item xs={12} md={10}>
+            <Card>
+              <CardHeader
+                className="gc360_header"
+                title={
+                  <Grid container alignItems="center" position="relative" justifyContent="center">
+                    {/* Badge positioned on the top-left corner */}
+                    <Badge
+                      badgeContent={foundReports.length}
+                      color="error"
+                      sx={{
+                        position: 'absolute',
+                        top: 15,
+                        left: 15,
+                        transform: 'translate(-50%, -50%)',
+                      }}
+                    />
+                    <Typography variant="h5" align="center">
+                      My Recently <span className={styles.yellowText}>Found</span> Items
+                    </Typography>
+                  </Grid>
+                }
+              />
+            </Card>
+          </Grid>
+          <Grid item xs={12} md={10}>
+            {/* Render header row */}
+            {reportHeader()}
+            {/* Filter and display found items */}
+            {foundReports
```

Comment:
I know it's rare to have two found items at once, but I think we should put a small gap between the reports on mobile screens, because it's difficult to tell the reports apart from one another.

<img width="300" alt="Screen Shot 2024-12-04 at 19 04 50" src="https://github.com/user-attachments/assets/57109c9e-2d6e-4eeb-a30f-6a01955fdf0e">

Reply:
I resolved this in my commits